os/drivers/i2c/Kconfig

```diff
@@ -39,4 +39,10 @@ config I2C_WRITEREAD
 	bool "Support the I2C writeread() method"
 	default n
 
+config I2C_ARTIK_EXTENSIONS
+	bool "Support for incomplete I2C transactions for Artik (S5J) chips"
+	depends on ARCH_CHIP_S5JT200
+	default y
+
+
 endif # I2C
```

Comment:
Only i2C protocol specific can be put here. This particulat config is chip specific. It can be moved to s5j configuration file